### ISSUE TITLE
Tests

### ISSUE THREAD

**Issue by djsutherland** [dougalsutherland/grunt-mathjax-node-page#1]
Is there a useful way to test this?

**Comment by jamesplease** [comment]
One way to test this would be to use the [grunt-contrib-uglify](https://github.com/gruntjs/grunt-contrib-uglify) approach, which works a little bit like this (it's been awhile, so I may be off a little bit):

1. they have a series of [test input files](https://github.com/gruntjs/grunt-contrib-uglify/blob/master/test/fixtures/src/comments.js)
2. they run those using the uglify CLI, with a certain set of options, to generate [expected outputs](https://github.com/gruntjs/grunt-contrib-uglify/blob/master/test/fixtures/expected/comments.js)
3. they have [grunt tasks](https://github.com/gruntjs/grunt-contrib-uglify/blob/2773e7ea1332d12bd17fed8f186aa196f04caea3/Gruntfile.js#L31) to run their plugin with different options, and the different source files, into a [gitignore'd temporary directory](https://github.com/gruntjs/grunt-contrib-uglify/blob/2773e7ea1332d12bd17fed8f186aa196f04caea3/.gitignore#L3)
4. they compare the grunt-generated output against the checked-in fixture data [within their test suite](https://github.com/gruntjs/grunt-contrib-uglify/blob/master/test/uglify_test.js#L71-L75)

**Comment by djsutherland** [comment]
Yeah, that makes sense. I probably don't care enough about this four-line thing to do that much work, though. :)

**Comment by jamesplease** [comment]
That’s reasonable 👍
